corehq/apps/hqadmin/management/commands/cchq_prbac_bootstrap.py

```diff
@@ -223,7 +229,8 @@ def remove_grant(self, priv_slug, dry_run=False):
     ]
 
     BOOTSTRAP_GRANTS = {
-        'community_plan_v0': community_plan_features,
+        'community_plan_v0': community_plan_v0_features,
+        'community_plan_v1': community_plan_v1_features,
```

Comment:
it's possible to create custom roles, so I'd recommend checking that there is no preexisting ```community_plan_v1``` role.

you could an assertion in the migration such as:
```assert not Role.objects.filter(slug='community_plan_v1').exists()```

Generally I'd expect the assertion to pass, but it would be good for the migration to fail hard on a production environment if that's not the case.